packages/actions-global/src/actions/handle-action-error.ts

```diff
@@ -9,15 +9,19 @@ import browserHistory from '@commercetools-frontend/browser-history';
 import showApiErrorNotification from './show-api-error-notification';
 import showUnexpectedErrorNotification from './show-unexpected-error-notification';
 
-type ActionError =
-  | Error
-  | {
-      statusCode: TStatusCode;
-      body: {
-        message: string;
-        errors?: TAppNotificationApiError | TAppNotificationApiError[];
-      };
-    };
+type SdkError = {
+  statusCode: TStatusCode;
+  body: {
+    message: string;
+    errors?: TAppNotificationApiError | TAppNotificationApiError[];
+  };
+};
+
+type ActionError = Error | SdkError;
+
+function isSdkError(error: ActionError): error is SdkError {
+  return (error as SdkError).body !== undefined;
```

Comment:
To avoid the casting we should also be able to do:

```ts
return ‘body’ in error && error.body !== undefined;
```

Reply:
They use casting in TS documentation, so I thought it's a more idiomatic way: https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates

But I personally don't mind to use `in`-syntax as well. WDYT?

Reply:
Yeah I don’t mind either. In other examples I found they use `in` operator (also documented below the section in the doc link).

I find it without the casting a bit more clear, but it’s fine either way. I just wanted to mention it